### ISSUE TITLE
`no-array-method-this-argument` doesn't warn about `Array.from()`

### ISSUE THREAD

**Issue by EvgenyOrekhov** [sindresorhus/eslint-plugin-unicorn#2144]
`no-array-method-this-argument` rule doesn't warn about `Array.from()`

```js
const foo = Array.from(bar, element => isUnicorn(element), baz);
```

Expected: ``Do not use the `this` argument in `Array#from()` `` warning in the above example for the `baz` argument.

Actual: no warning.
